lib/public/components/common/chart/structures/BarPropertiesProvider.js

```diff
@@ -0,0 +1,28 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+/**
+ * @callback ChartBarToChartBarProperty
```

Comment:
I understand why you do this, but normally the callback should not be so generic. It works here because all the input and results are the same (number or number[], never string?). But then you couple things that should not be...
Anyway, fine for now, but please remove the string from the return type